### DISCUSSION TITLE
Fix for "cannot import name 'cached_property'" build errors

### DISCUSSION THREAD

**Discussion by brockfanning** [open-sdg/open-sdg#1923] [Announcements]
One of the dependencies of the data build -- "frictionless" -- recently started requiring Python 3.8. Most implementations of Open SDG use Python 3.7, and so the data builds will fail. In order to fix this problem, you can "pin" to a prior version of frictionless. Add the following line to the `requirements.txt` file in your data repository:

```
frictionless==4.40.8
```

Note that the requirements.txt file is typically in the "scripts" folder. For example here is the file in our "data starter", with this fix already in place: https://github.com/open-sdg/open-sdg-data-starter/blob/develop/scripts/requirements.txt

**Comment by pixerize** [comment]
Thank you, Brock, you have saved me a lot of time :)

**Comment by nikolabul** [comment]
Thanks @brockfanning 👍
